### PR TITLE
fix: add black icon for pwa;

### PR DESCRIPTION
**Pull request by ilasw**
## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://mi-468-change-app-icon.preview.app.daily.dev

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]
Run them through https://tinypng.com/ to reduce the size about ~75%

**Review by omBratteng — CHANGES_REQUESTED** [review]
Run them through [tinypng.com](https://tinypng.com/) to reduce the size about ~75%

File: packages/webapp/public/android-chrome-192x192.png
Comment (nensidosari) [inline packages/webapp/public/android-chrome-192x192.png:1]: Just checking, do these need to be rounded again?

Comment (ilasw) [reply packages/webapp/public/android-chrome-192x192.png:1]: Seems to be rounded by device: 

iOS
image

Android
telegram-cloud-photo-size-4-5877356178856592093-x

**Review by nensidosari — APPROVED** [review]
Code looks good, but I'm not able to test through the preview domain. What is wrong with it?

**Comment by nensidosari** [comment]
> Don't know, I'm using this link to test it: https://daily-webapp-git-mi-468-change-app-icon-back-396b32-dailydotdev.vercel.app/

@ilasw but the app doesn't load with this link, I see just a blank page

**Comment by ilasw** [comment]
@nensidosari I know, but subdomain.daily.dev won't render :| It happened before for long branch names.
I can try to create a new short branch and see if it works. What do you think?

**Review by sshanzel — APPROVED** [review]
Looks amazing to me.

**Comment by nensidosari** [comment]
@ilasw no problem, not a big change here so not necessary. U can proceed with merging 🙌

**Comment by omBratteng** [comment]
> > Don't know, I'm using this link to test it: [daily-webapp-git-mi-468-change-app-icon-back-396b32-dailydotdev.vercel.app](https://daily-webapp-git-mi-468-change-app-icon-back-396b32-dailydotdev.vercel.app/)
> 
> @ilasw but the app doesn't load with this link, I see just a blank page

Seems like Vercel is stuck at generating SSL

**Comment by omBratteng** [comment]
I've manually changed the preview domain, so it should work now

**Comment by ilasw** [comment]
thanks @omBratteng ! Now it works 💯

**Pull request merged**